metaci/build/models.py

```diff
@@ -321,7 +326,21 @@ def run(self):
 
             # Initialize the project config
             project_config = self.get_project_config()
-
+            if self.plan.role == "push_sandbox" or self.plan.role == "push_production":
+                try:
+                    send_start_webhook(project_config, self.release, self.plan.role)
+                except Exception as err:
+                    message = (
+                        f"Error while sending implementation stop step webhook: {err}"
+                    )
+                    self.logger.error(message)
+                    set_build_info(
+                        build,
+                        status="error",
+                        exception=message,
+                        time_end=timezone.now(),
+                    )
+                    return
```

Comment:
@jofsky I don't think we need explicit error handling for this (lines 332-343); it should get caught by the exception handler on line 332.